ST_main.c

```diff
@@ -202,21 +223,35 @@ int main(int argc, char **argv) {
 
 			rgroup_PartResources();
 
+         //check_sizes("'main' after 'rgroup_PartResources'");
+
 #ifdef STEPWAT
 			if (!isnull(SXW.debugfile) ) SXW_PrintDebug(0);
 #endif
 
 			rgroup_Grow();
 
+      //check_sizes("'main' after 'rgroup_Grow'");
+
 			mort_Main(&killedany);
 
+      //check_sizes("'main' after 'mort_Main'");
+
 			rgroup_IncrAges();
 
+      //check_sizes("'main' after 'rgroup_IncrAges'");
+
       // Added functions for Grazing and mort_end_year as proportional killing effect before exporting biomass end of the year
 			grazing_EndOfYear();
-                        save_annual_species_relsize();
 
-			mort_EndOfYear();
+      //check_sizes("'main' after 'grazing_EndOfYear'");
+
+
+      save_annual_species_relsize();
+
+      		mort_EndOfYear();
+
+      //check_sizes("'main' after 'mort_EndOfYear'");
```

Comment:
@kpalmqui After you review the changes I made to correct issue #111 should we get rid of these check size calls?

Reply:
I think it is fine to remove most of the check_sizes calls, but let's retain the last check (commented out of course) at the end of the year.